src/scss/main.scss

```diff
@@ -27,7 +27,7 @@
 @import '../components/fieldset/fieldset';
 @import '../components/footer/footer';
 @import '../components/header/header';
-@import '../components/helpers/grid';
+@import '../components/helpers/grid/grid';
```

Comment:
I think we should either remove the helpers folder or move the grid files into foundations. Grid isn't really a component

Reply:
I agree I think one should go back to foundation as it's not a component. Reason I am saying this is not a component is that we don't expect users to use this grid macro but the grid classes.